charts/kong/README.md

```diff
@@ -725,6 +725,9 @@ section of `values.yaml` file:
 | userDefinedVolumes                      | Create volumes. Please go to Kubernetes doc for the spec of the volumes                                                                                  |                                    |
 | userDefinedVolumeMounts                 | Create volumeMounts. Please go to Kubernetes doc for the spec of the volumeMounts                                                                        |                                    |
 | terminationGracePeriodSeconds           | Sets the [termination grace period](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-execution) for Deployment pod | 30                                 |
+| serviceDiscovery.enabled                | Enables Kong instance service discovery                                                                                                                  |  false                             |
```

Comment:
> Enables Kong instance service discovery

as a user, after reading this, how can I tell if I want "service discovery" or not? how do I find the place where I will get that information?

suggested reword: "If false: the ingress controller will control a pre-determined set of Gateway instances based on Admin API URLs. If true: the ingress controller will dynamically locate Gateway instances by watching the specified Kubernetes service."

If too long, maybe make it more concise and point at the _The `serviceDiscovery` section_ section of the README.